doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,329 @@
+									               								   
+[Top](#top)		
+		
+		
+## 13. Access-Control-Allow-Methods
+
+It is a response HTTP header in Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-allow-methods).
+
+[Top](#top)
+
+									  
+## 14. Access-Control-Max-Age
+
+It is a response HTTP header in Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-max-age).
+		
+    Access-Control-Max-Age: 86400
+
+[Top](#top)
+
+
+## 15. Access-Control-Expose-Headers
+
+It is a response HTTP header in Orion.
+Its operation is related with CORS, see specific documentation at [../user/cors.md](https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/user/cors.md#access-control-expose-headers).
+	
+    Access-Control-Expose-Headers: Fiware-Correlator, Fiware-Total-Count, Location.
+
+[Top](#top)
+
+
+## 16. Allow
+
+Allow HTTP header is used in response notification.
+It notify that which methods are allowed by the specified host/resource.
+
+    Allow: GET
+
+[Top](#top)
+		
+    
+## 17. Location
+
+Location HTTP header is used in responses.
+Response contains a Location header which holds the subscription ID: a 24 digit hexadecimal number used for updating and cancelling the subscription.
+we use `-v` in requests to get the Location header in the response.
+	
+    curl -v localhost:1026/v2/subscriptions -s -S -H 'Content-Type: application/json' -d @- <<EOF
+    {
+     //payload
+    }
+    EOF
+	
+And the response that we will get:
+	
+    Location: /v2/subscriptions/57458eb60962ef754e7c0998
+	
+[Top](#top)
+
+
+## 18. Ngsiv2-AttrsFormat
+
+It is a response HTTP header in Orion.
+Notifications must include the `Ngsiv2-AttrsFormat` HTTP header with the value of the format of the associated subscription, 
+so that notification receivers are aware of the format without needing to process the notification payload.
+Note that if a custom payload is used for the notification then a value of `custom` is used for the `Ngsiv2-AttrsFormat` header
+in the notification.	
+Ngsiv2-AttrsFormat is a NON-MODIFIABLE header in custom notifications.
+Any attempt of doing so (e.g. `"httpCustom": { ... "headers": {"Ngsiv2-Attrsformat": "something"} ...}` will be ignored.
+	
+    Ngsiv2-Attrsformat: normalized
+
+[Top](#top)	
+
+
+## 19. Fiware-Service
+
+Fiware-Service is an optional header used in requests.
+When `-multiservice` is used Orion includes the `Fiware-Service` header in the notifyContextRequest and notifyContextAvailability request messages associated to subscriptions in the given tenant/service (except for the default service/tenant, in which case the header is not present).
```

Comment:
header in the notifyContextRequest and notifyContextAvailability request -> header in the notification requests

Reply:
Fixed in a427bd9